test/hw11/test_task1.py

```diff
@@ -0,0 +1,89 @@
+import pytest
+
+from All_home_works.hw11.task1 import SimplifiedEnum
+
+
+class ColorsEnum(metaclass=SimplifiedEnum):
+    RED = "RED"
+    BLUE = "BLUE"
+    ORANGE = "ORANGE"
+    BLACK = "BLACK"
```

Comment:
What is the minimum number of items in the enum to test this feature?

Reply:
To test it 1 item needed, but it's better to use at least 2 items.